cart.go

```diff
@@ -495,3 +473,31 @@ func mutateURL(original string, mutate bool) string {
 
 	return safe.String()
 }
+
+func httpClient(url, token string) []byte {
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	req.Header.Set("Circle-Token", token)
+	req.Header.Set("Accept", "application/json")
+
+	client := &http.Client{}
+	res, err := client.Do(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if res.StatusCode != http.StatusOK {
```

Comment:
This should probably just return an error, instead of panicking here.

Reply:
Fixed